examples/with-bitcoin/src/createBtcTx.ts

```diff
@@ -31,206 +27,194 @@ async function main() {
     defaultOrganizationId: process.env.ORGANIZATION_ID!,
   });
 
-  const cliPrompts = [
+  const ECPair = ECPairFactory(ecc);
+  const pair = ECPair.fromPublicKey(Buffer.from(publicKeyCompressed, "hex"));
+
+  // Only relevant for taproot.
+  const xOnlyPublicKey = pair.publicKey.slice(1, 33);
```

Comment:
probably not necessarily our duty to go super deep into the technicals of taproot/tweaking/etc (we'd expect our users to have an idea).

however if you have a reference doc that you really enjoyed using and thought was helpful, would be nice to add it here 😄

Reply:
That's not really the tweak part, it's simply the public key (untweaked). I think the reason why bitcoinjs wants the "x-only" pubkey is because the underlying secp library which does the tweaking also expects an "x-only" public key.

Reply:
Addressed in https://github.com/tkhq/sdk/pull/382/commits/61985b75fbee332b437c241fdb2c376a9e1c919b